sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/analysis/Analyzer.scala

```diff
@@ -687,7 +687,8 @@ class Analyzer(
         resolved
       } else {
         plan match {
-          case u: UnaryNode if !u.isInstanceOf[SubqueryAlias] =>
+          case u: UnaryNode
```

Comment:
How about `EvaluatePython`? What's the specific rule to blacklist a `UnaryNode`?